src/DurableTask.SqlServer.AzureFunctions/SqlMetricsProvider.cs

```diff
@@ -3,12 +3,15 @@
 
 namespace DurableTask.SqlServer.AzureFunctions
 {
+    using System;
     using System.Threading;
     using System.Threading.Tasks;
 
     public class SqlMetricsProvider
     {
         readonly SqlOrchestrationService service;
+        DateTime metricsTimeStamp = DateTime.MinValue;
+        SqlScaleMetric? metrics;
```

Comment:
Added this cache. If the target scaler calls `GetMetricsAsync` within 5 seconds, return the last result to make the target scaler remains the same decision as last time.